source/utilities.tex

```diff
@@ -1062,6 +1069,69 @@
 \tcode{*this}.
 \end{itemdescr}
 
+\indexlibrarymember{operator=}{pair}%
+\begin{itemdecl}
+template<@\exposconcept{pair-like}@ P> constexpr pair& operator=(P&& p);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+\begin{itemize}
+\item
+\tcode{\exposconcept{different-from}<P, pair>}\iref{range.utility.helpers}
+is \tcode{true},
+\item
+\tcode{remove_cvref_t<P>} is not a specialization of \tcode{ranges::subrange},
+\item
+\tcode{is_assignable_v<T1\&, decltype(get<0>(std::forward<P>(p)))>}
+is \tcode{true}, and
+\item
+\tcode{is_assignable_v<T2\&, decltype(get<1>(std::forward<P>(p)))>}
+is \tcode{true}.
+\end{itemize}
+
+\pnum
+\effects
+Assigns \tcode{get<0>(std::forward<P>(p))} to \tcode{first} and
+\tcode{get<1>(std::forward<P>(p))} to \tcode{sec\-ond}.
+
+\pnum
+\returns
+\tcode{*this}.
+\end{itemdescr}
+
+\indexlibrarymember{operator=}{pair}%
+\begin{itemdecl}
+template<@\exposconcept{pair-like}@ P> constexpr const pair& operator=(P&& p) const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+\begin{itemize}
+\item
+\tcode{\exposconcept{different-from}<P, pair>} is \tcode{true},
```

Comment:
```suggestion
\tcode{\exposconcept{different-from}<P, pair>}\iref{range.utility.helpers}
is \tcode{true},
```

Reply:
done. Some might say the reference in the previous itemdescr is sufficient tho, so this change (as well as any deviations from the paper for that matter) could still get rejected in the final merge.

Reply:
As the initial author, I'm fine with this change.
As a side note, i think we might want (as an editorial thing, after c++23) move `different-from` to utility so that it's easier to use from more places